core/src/main/java/org/elasticsearch/action/count/CountRequest.java

```diff
@@ -64,12 +62,12 @@
     @Nullable
     private String preference;
 
-    private BytesReference source;
-
     private String[] types = Strings.EMPTY_ARRAY;
 
     private int terminateAfter = DEFAULT_TERMINATE_AFTER;
 
+    private SearchSourceBuilder searchSourceBuilder = new SearchSourceBuilder();
```

Comment:
Going to still use SearchSourceBuilder internally for simplicity but will remove the setter and getter for SearchSourceBuilder so it isn't exposed to the API